modules/ROOT/partials/misc/using-community-lang-packs.adoc

```diff
@@ -1,17 +1,18 @@
 === Using the community language packs
 
-{% if forModuleLoaders == false %}
+ifeval::[{forModuleLoaders} == false]
 > *Note*: {companyname} recommends using the `language_url` for the community language packs, to avoid copying the language pack into the `tinymce/langs` folder every time you upgrade {productname}.
-{% endif %}
+endif::[]
 
 To use change the user interface language using a community language pack:
 
 . Download the language pack from link:{gettiny}/language-packages/[the {companyname} Community Language Packages download page].
-{% if forModuleLoaders == true %}
+ifeval::[{forModuleLoaders} == true]
 . Unzip and import/require the language file.
-{% else %}
+endif::[]
+ifeval::[{forModuleLoaders} == false]
```

Comment:
This doesn't have the same meaning, as if `forModuleLoaders` is not defined then this won't render whereas it would have previously. As such, this should use `!= true` instead.
```suggestion
ifeval::[{forModuleLoaders} != true]
```

Reply:
done